news/2024/2024-12-22-touhou-project-mania-cup-4-registrations-now-open.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Touhou Project Mania Cup 4: Registrations are Now Open!"
+date: 2024-12-22 12:00:00 +0000
+---
+
+![](/wiki/Tournaments/TMC/4th/image/banner@2x.png)
+
+With the celebration of the Touhou Project's 30th birthday, the Touhou Project Mania Cup is returning with its 4th iteration, bringing tons of improvement in competitiveness and player enjoyment!
+
+The [Touhou Project Mania Cup 4th](/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4k tournament. The tournament features 100% Touhou-related music and aims to bring top-tier competition to the tournament scene.
+
+We’ve had 80 teams [last year](/wiki/en/Tournaments/TMC/3rd), with **Bloxy cola** reaching the top with an undefeated performance. [This year](/wiki/en/Tournaments/TMC/4th), with more teams joining the competition, which team will rise to the challenge and become the best? 
+
+## What's new
+
+- **A Group Stage will debut this year** featuring a 16-group format. This means that we will have **64 teams** in the main event.
+  - If more than 70 teams play the qualifier, the group stage will use the split double elimination format.
+  - If fewer than 70 teams play the qualifier, the group stage will use the standard double elimination format.
+
+![Groups format](/wiki/Tournaments/TMC/4th/image/group-stage-bracket.png)
+
+  - **Each team will have 2 bans in the group stage**, providing extra strategic options throughout the different weeks.
+- **The "X" Picks will apply for RC and LN slots**, giving players more options to pick harder charts.
+- Two teams from each group will advance, with a total of 32 teams competing in a standard double-elimination bracket for the championship.
+
+## Custom charts 
+
+With almost **50 custom mappers** featured, this year we are bringing you a **nearly full custom** mappool from beginner to high-end. We hope this will not only enhance our Touhou journey but also contribute to the osu! community by adding more Touhou maps!
```

Comment:
```suggestion
With almost **50 custom mappers** contributing, this year we are bringing you mappools that are **almost entirely custom-made** with maps ranging from the lower end of the difficulty spectrum to the very top. We hope this will not only enhance our Touhou journey, but also contribute to the osu! community by adding more Touhou maps!
```